src/organisms/gv-documentation/gv-documentation.js

```diff
@@ -119,6 +119,9 @@ export class GvDocumentation extends LitElement {
           /* for Firefox */
           min-height: 0;
           padding: 0.5rem;
+
+          /* add padding when save bar is open */
+          padding-bottom: var(--gv-policy-studio--padding-save-bar);
```

Comment:
For var CSS we use Emmet Cheat sheet syntax https://docs.emmet.io/cheat-sheet/ 

`--gv-policy-studio--pb`

Reply:
haan i didn't know ok. but don't you think it's necessary to add an identifier to the subject (`save-bar`).  it's only linked to the save bar and  the padding bottom is very generic 💭 
maybe 
`--gv-policy-studio-save-bar--pb` ?

Reply:
No, I don't think, it's just to override padding-bottom, if we need this for other tricks, it should be okay

Reply:
okay